backend/src/data/column_indexes.py

```diff
@@ -1,24 +1,31 @@
+# TODO(asurite): add logic to convert number to column name, or other way around.
+# There are references to letters everywhere, which isn't good.
+# Maybe have this sheet be letters, because it corresponds better to reality?
+
 master = {
     'first': 1,
     'last': 0,
-    'id': 2,
-    'status': 3,
-    'email': 4,
-    'phone': 5,
-    'campus': 6,
-    'cohort': 7,
-    'mt': 8,
-    'committee': 9,
-    'leadership': 10,
-    'service': 11,
-    'civil-mil': 12,
-    'ols': 13,
-    'mt_attendance': 14,
-    'committee_attendance': 15,
-    'no_shows': 16,
-    'triggers_current': 17,
-    'triggers_last': 18,
-    'triggers_two_semesters': 19
+    'asurite': 2,
+    'id': 3,
+    'status': 4,
+    'email': 5,
+    'phone': 6,
+    'campus': 7,
+    'cohort': 8,
+    'mt': 9,
+    'committee': 10,
+    'leadership': 11,
+    'civil_mil': 12,
+    'hours_total': 13,
+    'service': 14,
+    'ngsc': 15,
+    'ols': 16,
+    'mt_attendance': 17,
+    'committee_attendance': 18,
+    'no_shows': 19,
+    'triggers_current': 20,
+    'triggers_last': 21,
+    'triggers_two_semesters': 22
 }
 
 engagement_responses = {
```

Comment:
@jseidne This will need to be updated to reflect how you changed the Responses tab, e.g. consolidating hours into only one thing. 

We don't need every field - we only pull the data that actually matters for visualizing on the website. I don't think you'll need to add any new fields. Instead, make sure the indices are still correct.

Don't rename 'id' to 'asurite' yet, I'll do that in a separate pull request.